package.json

```diff
@@ -104,7 +103,6 @@
     "gray-matter": "^4.0.3",
     "gunzip-maybe": "^1.4.2",
     "husky": "^8.0.0",
-    "jest": "^26.6.3",
```

Comment:
Let's remove remove `jest.config.js` and:

https://github.com/electron/website/blob/11a406efd3b82305e7a3dfd3e62845d0142bb5e2/.eslintrc.js#L8

Reply:
[1b0f806](https://github.com/electron/website/pull/560/commits/1b0f806f1001aa22bafacf98dd7df4dff7a0698b)